### DISCUSSION TITLE
i3 window title not disabling for Alacritty

### DISCUSSION THREAD

**Discussion by MorrisonWill** [i3/i3#5223] [Q&A] [answered]
I have the following in my i3 config to disable window titles entirely:
```
default_border pixel 1
default_floating_border pixel 1
```

This was working until I updated my system, and now the title is back for some windows. Here's an example with Firefox and Alacritty:

![screenshot-1666623086](https://user-images.githubusercontent.com/34083697/197556401-eedca1e2-8215-425d-88bf-eba6abc5f378.jpg)

Why is this happening? How can I remove that title?

Thanks

**Comment by orestisfl** [comment, marked as answer]
See https://github.com/i3/i3/issues/5149 and wait for the new bugfix release soon

**Reply by MorrisonWill** [reply]
Thanks for the response.

**Comment by ghost** [comment]
But in the release notes, you didn't mention the changelog for fixing the default behaviour of `default_border` configuration.

**Reply by orestisfl** [reply]
Which release notes? 4.21.1 includes the fix. We never really changed the behaviour of default_border explicitly.

**Reply by ghost** [reply]
Ofc that is what I was referring to. "Fix title bar disablement for certain applications when default_border pixel used".

**Reply by orestisfl** [reply]
I have no idea where you found this quote.

OP issue's related release note is "Motif hints: respect maximum border style configuration set by user".

**Reply by ghost** [reply]
My bad. Understood 😅
